### PR TITLE
Fix typo in Ethereum stack developer docs

### PR DESCRIPTION
**Pull request by saif-11bit**

### CODE REVIEW

File: src/content/developers/docs/ethereum-stack/index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ As a dapp developer, you'll need to write smart contracts only if you want to ad
 
 In order for an application to interact with the Ethereum blockchain, it must connect to an [Ethereum node](/developers/docs/nodes-and-clients/). Connecting to a node allows you to read blockchain data and/or send transactions to the network.
 
-Ethereum nodes are computers running software - an Ethereum client. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Ethereum nodes ARE the Ethereum blockchain. They collectively store the state of the Ethereum blockchain and reach consensus on transactions to mutate the blockchain state.
+Ethereum nodes are computers running software - an Ethereum client. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. Ethereum nodes are the Ethereum blockchain. They collectively store the state of the Ethereum blockchain and reach consensus on transactions to mutate the blockchain state.
```
Comment (minimalsm) [inline src/content/developers/docs/ethereum-stack/index.md:36]: I think this was deliberate to add emphasis here. However, thinking about translations (cc: @lukassim) I don't think it makes a ton of sense to keep it this way. But I also don't think removing the emphasis altogether is correct. What about if we wrap this sentence with bold tags?

```suggestion
Ethereum nodes are computers running software - an Ethereum client. A client is an implementation of Ethereum that verifies all transactions in each block, keeping the network secure and the data accurate. **Ethereum nodes are the Ethereum blockchain**. They collectively store the state of the Ethereum blockchain and reach consensus on transactions to mutate the blockchain state.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

File: src/content/developers/docs/ethereum-stack/index.md (hunk 1 above)
Comment (lukassim) [reply src/content/developers/docs/ethereum-stack/index.md:36]: Looking at the TM, most translators have kept this in uppercase, only a handful of translators have changed it to lowercase.
Bold tags should work fine in Crowdin, so I guess either option would be okay for the translations.

**Review by minimalsm — APPROVED** [review]

**Comment by minimalsm** [comment]
Thanks for the PR @saif-11bit 💪

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And remember to [claim your POAP](https://ethereum.org/en/contributing/#poap) also 🥳! We're currently working on our 2022 POAPs and they'll be released soon.

**Comment by minimalsm** [comment]
@all-contributors please add @saif-11bit for docs

**Pull request merged**